assets/gogs.css

```diff
@@ -149,6 +149,9 @@ code.wrap {
 .ui.segment {
   box-shadow: none;
 }
+.ui .text.err {
```

Comment:
Since this is a direct copy of the class below, lets use what's already available.

Please remove this class again and we'll just use the `.ui .text.red` class instead.